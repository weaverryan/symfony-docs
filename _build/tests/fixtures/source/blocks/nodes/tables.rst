
==========  ========================================  ==========================================
Route path  If the requested URL is /foo              If the requested URL is /foo/
==========  ========================================  ==========================================
/foo        It matches (200 status response)          It doesn't match (404 status response)
/foo/       It makes a 301 redirect to /foo/          It matches (200 status response)
==========  ========================================  ==========================================

+--------+------------+
| Cell 1 | Cell 2     |
+--------+------------+
| Cell 3 | Cell 4     |
+--------+------------+
| Cell 5 | Cell 6     |
|        | extra line |
+--------+------------+

+--------+------------+
| Cell 1 | Cell 2     |
+========+============+
| Cell 3 | Cell 4     |
+--------+------------+
| Cell 5 | Cell 6     |
|        | extra line |
+--------+------------+

+--------+---------------+
| Cell 1 | Cell 2        |
+========+===============+
| Cell 3 | Cell 4        |
+--------+---------------+
| Cell 5 | - List item 1 |
|        | - List item 2 |
|        | - List item 3 |
|        | - List item 4 |
+--------+---------------+
| Cell 7 | Cell 8        |
+--------+---------------+
