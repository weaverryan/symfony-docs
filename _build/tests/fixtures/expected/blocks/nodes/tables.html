<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<table class="docutils" border="1">
    <thead valign="bottom">
    <tr class="row-odd">
        <th class="head">Route path</th>
        <th class="head">If the requested URL is /foo</th>
        <th class="head">If the requested URL is /foo/</th>
    </tr>
    </thead>
    <tbody valign="top">
    <tr class="row-even">
        <td>/foo</td>
        <td>It matches (200 status response)</td>
        <td>It doesn't match (404 status response)</td>
    </tr>
    <tr class="row-odd">
        <td>/foo/</td>
        <td>It makes a 301 redirect to /foo/</td>
        <td>It matches (200 status response)</td>
    </tr>
    </tbody>
</table>
<table class="docutils" border="1">
    <tbody valign="top">
    <tr class="row-odd">
        <td>Cell 1</td>
        <td>Cell 2</td>
    </tr>
    <tr class="row-even">
        <td>Cell 3</td>
        <td>Cell 4</td>
    </tr>
    <tr class="row-odd">
        <td>Cell 5</td>
        <td>Cell 6
extra line</td>
    </tr>
    </tbody>
</table>
<table class="docutils" border="1">
    <thead valign="bottom">
    <tr class="row-odd">
        <th class="head">Cell 1</th>
        <th class="head">Cell 2</th>
    </tr>
    </thead>
    <tbody valign="top">
    <tr class="row-even">
        <td>Cell 3</td>
        <td>Cell 4</td>
    </tr>
    <tr class="row-odd">
        <td>Cell 5</td>
        <td>Cell 6
extra line</td>
    </tr>
    </tbody>
</table>
<table class="docutils" border="1">
    <thead valign="bottom">
    <tr class="row-odd">
        <th class="head">Cell 1</th>
        <th class="head">Cell 2</th>
    </tr>
    </thead>
    <tbody valign="top">
    <tr class="row-even">
        <td>Cell 3</td>
        <td>Cell 4</td>
    </tr>
    <tr class="row-odd">
        <td>Cell 5</td>
        <td>
            <ul>
                <li>List item 1</li>
                <li>List item 2</li>
                <li>List item 3</li>
                <li>List item 4</li>
            </ul>
        </td>
    </tr>
    <tr class="row-even">
        <td>Cell 7</td>
        <td>Cell 8</td>
    </tr>
    </tbody>
</table>
</body>
</html>
